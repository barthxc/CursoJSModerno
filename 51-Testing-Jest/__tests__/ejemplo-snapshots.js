const cliente = {
    nombre:'Pablo 2',
    balance: 500,
    tipo: 'Premium'
}

describe('Testing al cliente', ()=>{
    test('Es Pablo', () =>{
        expect(cliente).toMatchSnapshot();
    })
})

// npm t -- -u para actualizar tu objeto o eliminar un snapshot para crear lo de nuevo usando : npm t