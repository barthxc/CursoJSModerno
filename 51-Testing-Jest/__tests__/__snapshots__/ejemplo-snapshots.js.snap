// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Pablo 1`] = `
{
  "balance": 500,
  "nombre": "Pablo 2",
  "tipo": "Premium",
}
`;
